packages/salesforcedx-utils-vscode/src/index.ts

```diff
@@ -41,3 +41,4 @@ export {
   getRootWorkspacePath,
   getRootWorkspaceSfdxPath
 } from './workspaces';
+export { SETTING_CLEAR_OUTPUT_TAB, SFDX_CORE_CONFIGURATION_NAME, SFDX_CORE_EXTENSION_NAME } from './constants';
```

Comment:
the imports need to be alphabetized.  i'm surprised the linter didn't catch this.

Since this is being imported from `./constants`, it should be moved to line 25, before the import from `./context/...`